sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -385,6 +423,29 @@ protected ServiceBusProcessor()
             }
         }
 
+        private void ValidateErrorHandler()
+        {
+            if (_processErrorAsync == null)
+            {
+                throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture, Resources1.CannotStartEventProcessorWithoutHandler, nameof(ProcessErrorAsync)));
+            }
+        }
+
+        private void ValidateMessageHandler()
+        {
+            if (IsSessionProcessor)
+            {
+                if (_processSessionMessage == null)
+                {
```

Comment:
nit: We can merge these if conditions.

Reply:
We wouldn't want to go into the else block if it is a session processor that has _processSessionMessage populated. We should only go to the else block if not a session processor.